RELEASE-CHECKLIST.md

```diff
@@ -0,0 +1,28 @@
+# Releasing `substrate-archive`
+
+Here's how to make a new release of `substrate-archive`.
+
+## Checklist
+
+1. Make a new branch, name it whatever you want, but `vx.y.z-prep` is a good template
+1. Update `Cargo.toml` in `substrate-archive` and `substrate-archive-backend` with the new version number.
+1. Update all references to `substrate` crates and `polkadot` to their respective latest releases.
+1. Update the `CHANGELOG` as specified [here](https://keepachangelog.com/en/1.0.0/).
+1. Run all tests: `TEST_DATABASE_URL="postgres://localhost:5432/archive cargo test --all`.
```

Comment:
Do we assume the maintainer knows what is going on here with setting up the postgres DB?

Reply:
yeah, there's a wiki for that https://github.com/paritytech/substrate-archive/wiki/3.)-Configuring-Postgres-for-Archive

but maybe add a link to that or something